react/components/UI/Head/components/TwitterCard/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Head from 'react/components/UI/Head';
+
+const TwitterCard = ({ children: images }) => (
+  <Head>
+    <meta name="twitter:card" content="summary_large_image" />
+    {images.length > 0 && images.map(image => (
+      <meta property="og:image" content={image} />
+    ))}
+  </Head>
+);
```

Comment:
Why not just pass in an actual image here instead of having this logic to find one in it?

Reply:
Sorry, did not mean to check this in, WIP from another branch

Reply:
Or there's a reason that we have summary_large_image without image?